### PR TITLE
Bug fix

### LINKED ISSUE #116: [PE-D] view date  does not work when it is on the uncompleted tab
view 2021-04-20 shows 4 tasks even though only 1 of the task has the date 2021-04-20 in the uncompleted tab. 

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: rjeez/ped#5

### PR DESCRIPTION
**Pull request by hengyongming**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ Format: `edit INDEX [n/NAME] [desc/DESCRIPTION] [date/DATE] [s/STATUS] [t/TAG]
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
 * You can remove all the task’s tags by typing `t/` without specifying any tags after it.
+* There are 3 Task statuses: `expired`, `uncompleted` and `completed`. However you can cannot directly modify task's status to `expired`
```
Comment (noelmathewisaac) [inline docs/UserGuide.md:127]: Should we also add explanation for what each of the statuses mean? Also there's a space missing in line 132

Comment (hengyongming) [reply docs/UserGuide.md:127]: I mean completed and uncompleted is self-explanatory. The missing space I will add it next time

Comment (noelmathewisaac) [reply docs/UserGuide.md:127]: Ok can

**Review by noelmathewisaac — APPROVED** [review]
LGTM

**Review by CSmortal — APPROVED** [review]
LGTM

**Pull request merged**